.gitmodules

```diff
@@ -4,8 +4,8 @@
 	branch = main
 [submodule "src/code.cloudfoundry.org/bbs"]
 	path = src/code.cloudfoundry.org/bbs
-	url = https://github.com/cloudfoundry/bbs
-	branch = main
+	url = https://github.com/dimitardimitrov13/bbs
+	branch = CFAR-929_VCAP_SERVICES
 [submodule "src/code.cloudfoundry.org/auctioneer"]
```

Comment:
This looks to me like an unwanted change here

Reply:
Yes, you're right. I need to run the Diego Release test suite first. After it passes, the modules will be reverted.